Cargo.toml

```diff
@@ -22,19 +12,23 @@ license = "MIT"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-egui = { version = "0.29.1", default-features = false }
-
-# Used to fetch user folders
-directories = "5.0"
-
-# Used to fetch disks
-sysinfo = { version = "0.32", default-features = false, features = ["disk"] }
-
-# Used for persistent storage
-serde = { version = "1", features = ["derive"], optional = true }
-
-# Used to canonicalize paths
-dunce = "1.0.5"
+egui = { version = "0.29.1", default-features = false } # ui
+directories = "5.0" # fetch user folders
+dunce = "1.0.5" # canonicalize paths
+sysinfo = { version = "0.32", default-features = false, features = [
+    "disk",
+] } # fetch disks
+serde = { version = "1", features = [
+    "derive",
+], optional = true } # persistent storage
```

Comment:
I think having the comments above the dependencies is easier to read than behind them. Especially for those that are split across multiple lines. Could you change that?